lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -52,53 +52,49 @@ class Core
+    ["-h", "--help"]            => [ false, "Help banner."                                   ],
+    ["-k", "--kill"]            => [ true,  "Terminate the specified thread ID."             ],
+    ["-K", "--kill-all"]        => [ false, "Terminate all non-critical threads."            ],
+    ["-i", "--info"]            => [ true,  "Lists detailed information about a thread."     ],
+    ["-l", "--list-background"] => [ false, "List all background threads."                   ],
+    ["-v", "--verbose"]         => [ false, "Print more detailed info.  Use with -i and -l"  ])
 
   @@tip_opts = Rex::Parser::Arguments.new(
-    "-h" => [ false, "Help banner."                                   ])
+    ["-h", "--help"] => [ false, "Help banner."                                   ])
 
   @@debug_opts = Rex::Parser::Arguments.new(
-    "-h" => [ false, "Help banner."                                   ],
-    "-d" => [ false, "Display the datastore information."             ],
-    "-c" => [ false, "Display command history."                       ],
-    "-e" => [ false, "Display the most recent Error and Stack Trace." ],
-    "-l" => [ false, "Display the most recent logs."                  ],
-    "-v" => [ false, "Display versions and install info."             ],
-    "-s" => [ false, "Display database statistics."                   ])
+    ["-h", "--help"]            => [ false, "Help banner."                                   ],
+    ["-d", "--datastore"]       => [ false, "Display the datastore information."             ],
+    ["-c", "--command-history"] => [ false, "Display command history."                       ],
+    ["-e", "--error-trace"]     => [ false, "Display the most recent Error and Stack Trace." ],
+    ["-l", "--logs"]            => [ false, "Display the most recent logs."                  ],
+    ["-v", "--version"]         => [ false, "Display versions and install info."             ],
+    ["-s", "--stats"]           => [ false, "Display database statistics."                   ])
 
   @@connect_opts = Rex::Parser::Arguments.new(
-    "-h" => [ false, "Help banner."                                   ],
-    "-p" => [ true,  "List of proxies to use."                        ],
-    "-C" => [ false, "Try to use CRLF for EOL sequence."              ],
-    "-c" => [ true,  "Specify which Comm to use."                     ],
-    "-i" => [ true,  "Send the contents of a file."                   ],
-    "-P" => [ true,  "Specify source port."                           ],
-    "-S" => [ true,  "Specify source address."                        ],
-    "-s" => [ false, "Connect with SSL."                              ],
-    "-u" => [ false, "Switch to a UDP socket."                        ],
-    "-w" => [ true,  "Specify connect timeout."                       ],
-    "-z" => [ false, "Just try to connect, then return."              ])
-
-  @@search_opts = Rex::Parser::Arguments.new(
```

Comment:
Has this been moved somewhere else or was it not being used?

Reply:
The `@@search_opts` in here were not used, as far as I'm aware. Instead, the `@@search_opts` from `lib/msf/ui/console/command_dispatcher/modules.rb` is used. I'll double-check though 👍